source/instructors/setupcourses/export-course-data.rst

```diff
@@ -164,9 +164,39 @@ Export course sources to obtain a zip file containing all the currently publishe
 
    .. image:: /img/source_exportlinks.png
       :alt: Course Export Links
-      
 
+.. _course-blueprint-transfer:
+
+Course Blueprint Transfer
+-------------------------
+
+You can export course Blueprints to a file to transfer them to a different course.
+
+   1. 1. Navigate to the **Courses** page and open the course from which you want to export **Blueprints**.
+   2. Click the **Export** tab and then scroll down to **Course Blueprint Transfer**.
+   3. Click the **Export** button.
+   4. In the pop-up dialog, click **Export** again to download a JSON file containing the course Blueprints.
+   
+
+.. image:: /img/export_blueprints.png
+      :alt: Export Blueprints window
+
+With the exported file, you can share Blueprints with other instructors or import them into a different course.
+
```

Comment:
I think it would be helpful to have titles at the top of each section: Exporting and Importing